templates/lxd/_form-lxd-install.html

```diff
@@ -26,7 +26,7 @@ <h3 class="p-heading--5" id="modal-description">How should we get in touch?</h3>
                   <input class="p-checkbox__input" value="yes" aria-labelledby="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
                   <span class="p-checkbox__label" id="canonicalUpdatesOptIn">I agree to receive information about Canonical's products and services.</span>
                 </label>
-                <p>By submitting this form, I confirm that I have read and agree to Canonical's <a href="/legal/terms-and-policies/privacy-policy">Privacy Policy</a>.</p>
+                <p>By submitting this form, I confirm that I have read and agree to <a href="https://www.ubuntu.com/legal/dataprivacy">Canonical's Privacy Policy</a>.</p>
```

Comment:
@mtruj013 is there a particular reason why you used `/dataprivacy` instead of `/data-privacy`?

Reply:
No, I took it from big-data-and-ai form doc (since we don't seem to have one for this form). Didn't realize it was a redirected link though, updated